code/modules/research/techweb/all_nodes.dm

```diff
@@ -1187,6 +1187,7 @@
 		"crewconsole",
 		"idcard",
 		"libraryconsole",
+		"barcode_scanner",
```

Comment:
ABC order (bank machine is screwed up too)

Reply:
I wanted this to show up next to the library console since they go hand in hand.